packages/ibm-products/src/components/TagSet/TagSet.stories.jsx

```diff
@@ -133,11 +132,24 @@ export default {
   component: TagSet,
   tags: ['autodocs'],
   parameters: {
-    // docs: { page: mdx },
     styles,
     percy: {
       waitForTimeout: 1000,
     },
+    page: () => (
+      <StoryDocsPage
+        altGuidelinesHref={[
+          {
+            href: 'https://carbondesignsystem.com/components/tag/usage/',
+            label: 'Carbon Tag usage guidelines',
+          },
+          {
+            href: 'https://react.carbondesignsystem.com/?path=/docs/components-tag--overview',
+            label: 'Carbon Tag documentation',
+          },
+        ]}
+      />
+    ),
```

Comment:
```suggestion
    docs: {
      page: () => (
        <StoryDocsPage
          altGuidelinesHref={[
            {
              href: 'https://carbondesignsystem.com/components/tag/usage/',
              label: 'Carbon Tag usage guidelines',
            },
            {
              href: 'https://react.carbondesignsystem.com/?path=/docs/components-tag--overview',
              label: 'Carbon Tag documentation',
            },
          ]}
        />
      ),
    },
```

I think we missed a `docs:` here.